main/nginx/APKBUILD

```diff
@@ -310,7 +312,8 @@ _module() {
 
 	local soname; for soname in $sonames; do
 		mv "$pkgdir"/$_modules_dir/$soname ./$_modules_dir/$soname
-		echo "load_module \"modules/$soname\";" > ./etc/nginx/modules/$name.conf
+		rm -f "$pkgdir"/$_modules_dir/${soname}.old
```

Comment:
What’s the purpose of this? There’s no file with suffix `.old` generated, is it?

Reply:
Because we build the geoip2 module twice (once for http and once for stream) and since the module puts out both `so`s every time, the second build results in the first set of `so`s to be renamed `*.old`.

Building it twice is a bit of a hack, but it ensures that we can have two separate packages, one for the http version and one for the stream version, each package with the respective `so`